autoload/go/def.vim

```diff
@@ -15,7 +15,9 @@ endf
 " modified and improved version of vim-godef
 function! go#def#Jump(...)
 	if !len(a:000)
-		let arg = "-o=" . go#util#OffsetCursor()
+		" gives us the offset of the word, basically the position of the word under
+		" the cursor
+		let arg = s:getOffset()
```

Comment:
I think this comes from a wrong merge. Can you just rebase and squash your commits into one commit? That would fix these kind of issues. I can then re-check your PR, as right now I can't use it.